database/init.sql

```diff
@@ -0,0 +1,36 @@
+DROP TABLE IF EXISTS account;
+DROP TABLE IF EXISTS post;
+
+CREATE TABLE account (
+  id SERIAL PRIMARY KEY,
+  email TEXT UNIQUE NOT NULL,
+  password TEXT NOT NULL
+);
+
+CREATE TABLE account_role (
+  email TEXT NOT NULL,
+  role TEXT NOT NULL,
+  PRIMARY KEY (email, role),
+  FOREIGN KEY (email) REFERENCES account (email) ON UPDATE CASCADE ON DELETE CASCADE
+);
+
+CREATE INDEX account_role_email_index ON account_role (email);
+
+CREATE TABLE profile (
```

Comment:
Name profile looks like to be part of standard sql namespace. To keep up with consistency we should rename it to account_profile, analog to account_role

Reply:
okay